### PR TITLE
Add desktop version using media queries

### PR DESCRIPTION
**Pull request by braincee**
**_In this pull request, the following requirements were implemented:_**

- I added a desktop version to my portfolio website making it responsive to all on-screen devices.
- I added changes to the desktop version with media queries making it a min-width of 992px.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -584,4 +590,41 @@ i {
...
+    padding: 10px 11.25rem 0;
+  }
+
+  .container-5 {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 767px) {
+  .title-4 {
```
Comment (micheaol) [inline style.css]: Please ensure that your media queries for the Desktop breakpoint in the CSS file. (the value for `min-width` property should be in range of `992px` - `1100px`)

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @braincee , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @braincee,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by braincee** [comment]
Thanks very much, @micheaol and @TedLivist 

**_Note: Please I have implemented all the required and optional changes._**

thank-you-thanks

**Pull request merged**